Assets/Scripts/StandardSamples/UI/Storage/Player/UIPlayerDataStorageMenu.cs

```diff
@@ -75,8 +75,8 @@ public class UIPlayerDataStorageMenu : MonoBehaviour, ISampleSceneUI
         public GameObject UIFirstSelected;
 
         private string currentSelectedFile = string.Empty;
+        private Dictionary<string, string> localFileNameToContents = new Dictionary<string, string>();
```

Comment:
If I'm not mistaken, this data already exists within the Service class. Is there a reason to keep an independent copy of that data outside of it? Seems like it opens the possibility for there to be an inconsistency between the two.